Src/Plugins/Security/SEC101_046.SqlConnectionStringValidator.cs

```diff
@@ -66,7 +77,7 @@ public static string IsValidDynamic(ref string fingerprint, ref string message)
 
             string host, database, account, password;
 
-            if (groups.ContainsKey("host") && groups.ContainsKey("database") && groups.ContainsKey("account") && groups.ContainsKey("password"))
+            if (groups.ContainsKey(HostKey) && groups.ContainsKey("database") && groups.ContainsKey("account") && groups.ContainsKey("password"))
```

Comment:
>HostKey [](start = 35, length = 7)

if we are going to do this, we could use Fingerprint.HostKey and apply this for  the other strings as well.

Reply:
Complexity not necessary.

---
In reply to: [575576409](https://github.com/microsoft/sarif-pattern-matcher/pull/189#discussion_r575576409) [](ancestors = 575576409)